loopy/symbolic.py

```diff
@@ -2009,17 +2029,22 @@ def get_access_range(domain, subscript, assumptions=None, shape=None,
 
 class BatchedAccessRangeMapper(WalkMapper):
 
-    def __init__(self, kernel, var_names, overestimate=None):
+    def __init__(self, kernel, var_names, overestimate=False):
         self.kernel = kernel
         self.var_names = set(var_names)
         from collections import defaultdict
-        self.access_ranges = defaultdict(lambda: None)
+        self.access_maps = defaultdict(lambda: defaultdict(lambda: None))
         self.bad_subscripts = defaultdict(list)
+        self.overestimate = overestimate
```

Comment:
Underscore the ones not meant to be world-visible?

Reply:
Sure! e71636c406665a6f0f03ed3501148b3fa9f1254f